packages/fxa-auth-server/lib/senders/emails/templates/lowRecoveryCodes/index.stories.ts

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { Meta } from '@storybook/html';
-import { Template, storyWithProps } from '../../storybook-email';
+import { storyWithProps } from '../../storybook-email';
```

Comment:
@jodyheavener We should probably also remove unused imports on save with eslint when we do that prettier follow up. I'm also not sure why `buildStory` wasn't erroring in the other story files I fixed 🤔

Reply:
Oh what the heck, I did a find and replace for all the instances of `buildStory`. I wonder how they got in...

Reply:
Ohhh I think I had a new template PR simultaneously open with the Storybook PR, so I made the storybook change but didn't update it in my template PR after it got merged.